dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,66 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseOption):
+    def __init__(self):
+        """
+        Constructs the TextProfilerOption object with default values.
+
+        :ivar is_case_sensitive: option set for case sensitivity.
+        :vartype is_case_sensitive: bool
+        :ivar stop_words: option set for stop words.
+        :vartype stop_words: Union[None, list(str)]
+        :ivar words: option set for word update.
+        :vartype words: BooleanOption
+        :ivar vocab: option set for vocab update.
+        :vartype vocab: BooleanOption
+        """
+        self.is_case_sensitive = True
+        self.stop_words = None
+        self.words = BooleanOption(is_enabled=True)
+        self.vocab = BooleanOption(is_enabled=True)
+
+    def is_prop_enabled(self, prop):
+        """
+        Checks to see if a property is enabled or not and returns boolean.
+
+        :param prop: The option to check if it is enabled
+        :type prop: String
+        :return: Whether or not the property is enabled
+        :rtype: Boolean
+        """
+        return BaseColumnOptions.is_prop_enabled(self, prop)
+
+    def _validate_helper(self, variable_path='TextProfilerOptions'):
```

Comment:
instead of referencing `BaseColumnOptions`, lets just rename it so it is universal.
e.g. `BaseInspectorOptions` (the naming convention that will come to pass)
or for now could say `BaseProfilerOptions` if you don't want to say `inspector` yet